sbol/location.py

```diff
@@ -1,6 +1,7 @@
 from .identified import Identified
 from .constants import *
-from .property import URIProperty, ReferencedObject, LiteralProperty, OwnedObject
+from .property import URIProperty, ReferencedObject, LiteralProperty, OwnedObject, \
+                      IntProperty
```

Comment:
I'd much rather see two `from .property import` lines than see a backslash. And then I wonder why they aren't alphabetized. And truth be told I prefer them each on a separate line at that point.

```suggestion
from .property import IntProperty 
from .property import LiteralProperty
from .property import OwnedObject
from .property import ReferencedObject
from .property import URIProperty
```